privacy.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: privacy
+title: Privacy Notice
+---
+
+# Foreman Privacy Policy
+
+This document describes the data Foreman collects about you, why it is
+collected, what is done with it, and your options for how it's processed.
+
+The Foreman project does not share data with 3rd parties, except for hosting
+requirements (e.g. GitHub, YouTube) as detailed below, or when required to by
+law.
+
+# Types of data we collect
+
+* Contact details
+  * Name
+  * Email address
+* Data that identifies you
+  * IP addresses
+  * Browser UserAgent strings
+  * Debug archives
+  * Answers to survey questions
+
+# How and why we use your data
+
+## The website
+
+The Foreman website is a static site which does not track, collect, store
+information or set cookies. There is no account for the website itself. The
+webserver (Apache) logs will contain IP addresses, timestamps, and UserAgent
+strings - these are log-rotated and deleted after 4 weeks.
+
```

Comment:
We have Google analytics enabled for the site.

Reply:
We don't actually use it at the moment, I can't recall the last time I looked at it. we should disable it. And Google+ integration ;)

Reply:
well... I used it last week for someone asking how many IE users we have, so YMMV

Reply:
sorry, only saw this after I finished my updates, lag--. I'm honestly unsure of the privacy impact here, I saw some of the discussion around google analytics and it made me quite uncomfortable. If you can draft something to include here I'll be happy to include it, but this has to go out today / tomorrow so I can include it in the newsletter.